index.js

```diff
@@ -3,15 +3,15 @@ const got = require('got');
 const registryUrl = require('registry-url');
 
 function get(keyword, options) {
-	if (typeof keyword !== 'string') {
-		return Promise.reject(new TypeError('Keyword must be a string'));
+	if (typeof keyword !== 'string' && !Array.isArray(keyword)) {
+		return Promise.reject(new TypeError('Keyword must be either a string or an array of strings'));
 	}
 
 	if (options.size < 1 || options.size > 250) {
 		return Promise.reject(new TypeError('Size option must be between 1 and 250'));
 	}
 
-	keyword = encodeURIComponent(keyword);
+	keyword = encodeURIComponent(keyword).replace('%2C', '+');
```

Comment:
Is this to »clever«? 😎

Reply:
Why are you doing it? From the docs:

> separate multiple keywords with commas

So that should be `+` => `,`, but have you tested whether it supports encoded comma or not?

Reply:
The docs are wrong (Going to create a PR to fix this).
Keywords should be separated with a `+`. See https://github.com/yeoman/yo/issues/544#issuecomment-340901909

Compare:
https://registry.npmjs.org/-/v1/search?text=keywords:gulpplugin+sass+css&size=1
➡️ `"total": 38`

With:
https://registry.npmjs.org/-/v1/search?text=keywords:gulpplugin,sass,css&size=1
➡️ `"total": 14203`

The latter has even more results than searching for https://registry.npmjs.org/-/v1/search?text=keywords:gulpplugin&size=1 alone

Reply:
Ok, it seems like combining keywords with `,` acts like a `||` while combining them with `+` acts like a `&&`.

Reply:
Guess I have better examples over there: https://github.com/npm/registry/pull/235